### ISSUE TITLE
resolution options

### ISSUE THREAD

**Issue by travisfw** [dozingcat/AsciiCam#5]
I couldn't find a text editor on my phone that allowed for a small enough font to view the image. It should be possible to downsample to half resolution at least.

**Comment by ahmdshbz** [comment]
I agree. A lower resolution combined with wider range of characters (instead of the currently limited #, O, o, :, ., etc.) would be nice to have. 
The wider character range should help achieve reasonable picture quality with a much lower resolution.
